lib/embedded_ansible.rb

```diff
@@ -60,13 +61,19 @@ def self.start
   end
 
   def self.stop
-    return if MiqEnvironment::Command.is_container?
-    services.each { |service| LinuxAdmin::Service.new(service).stop }
+    if MiqEnvironment::Command.is_container?
+      container_stop
+    else
+      services.each { |service| LinuxAdmin::Service.new(service).stop }
```

Comment:
If I went with this rename I would probably go with `appliance_stop` and `appliance_disable` to match `appliance_start` which does more than start services in some cases. What do you think?

It's all private interface anyway so we could always change in a follow up.